recipe/build.sh

```diff
@@ -1,18 +1,10 @@
-#!/bin/bash
```

Comment:
Again, probably best to use the recipe from `defaults`.  If you make this change and the change for `bld.bat`, you can remove the `setup.py` from this recipe.

```

# necessary because conda symlinks
unlink $PREFIX/bin/conda
unlink $PREFIX/bin/activate
unlink $PREFIX/bin/deactivate

export CONDA_DEFAULT_ENV=''
$PYTHON conda.recipe/setup.py install

mkdir -p $PREFIX/etc/fish/conf.d/
cp $SRC_DIR/shell/conda.fish $PREFIX/etc/fish/conf.d/
```